shark/iree_utils/vulkan_utils.py

```diff
@@ -78,6 +78,8 @@ def get_vulkan_target_triple(device_name):
         triple = f"turing-rtx6000-{system_os}"
     elif all(x in device_name for x in ("RTX", "8000")):
         triple = f"turing-rtx8000-{system_os}"
+    elif all(x in device_name for x in ("TITAN", "RTX")):
```

Comment:
NIT: Can we switch up the RTX and TITAN, for aesthetic and consistency in the code?

Otherwise LGTM :)